bindings/bind_py_module.cc

```diff
@@ -38,21 +38,27 @@ PYBIND11_MODULE(_rascal, mod) {
 
   py::module m_rpr_mng = mod.def_submodule("RepresentationManager");
   m_rpr_mng.doc() = "Representation Manager Classes";
-  py::module m_feat_mng = mod.def_submodule("FeatureManager");
-  m_feat_mng.doc() = "Feature Manager Classes";
+  py::module m_models = mod.def_submodule("Models");
```

Comment:
Camelcase names for modules break Python conventions; these should be `"models"` and `"kernels"` (and `"representation_manager"` above)

Reply:
Speaking of which, shouldn't it be renamed to `representation_calculator` (with the doc accordingly changed)?

Reply:
the name of these submodules is meant to keep the beginning of the classes it binds so RepresentationManagerSortedCoulomb becomes RepresentationManager.SortedCoulomb
I am not sure what you are referring to about python convention.